### PR TITLE
Ana/add links to notifications

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Something I should have done in the Team/Notifications PR 😅 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/notifications/PageNotifications.vue [hunk 1]
```diff
@@ -16,10 +16,15 @@
...
 
-    <div
+    <router-link
       v-for="notification in notifications"
       :key="notification.id"
       class="notification"
+      title="Transactions"
+      :to="{
+        name: 'transactions',
+        params: { networkId: notification.link.split('/')[1] },
```
Comment (faboweb) [inline src/components/notifications/PageNotifications.vue:26]:
```suggestion
        params: { networkId: notification.link.trim('/') },
```

Comment (Bitcoinera) [reply src/components/notifications/PageNotifications.vue:26]: It doesn't work

Comment (faboweb) [reply src/components/notifications/PageNotifications.vue:26]: upsi a I see know

Comment (faboweb) [reply src/components/notifications/PageNotifications.vue:26]: why not `:to="notification.link"`?

Comment (faboweb) [reply src/components/notifications/PageNotifications.vue:26]: the logic you are using would only work for transaction events, am I wrong?

Comment (Bitcoinera) [reply src/components/notifications/PageNotifications.vue:26]: > why not :to="notification.link"?

It is the first thing I did but then we get the uncaught promise error on navigation. We need to set name and params

Comment (faboweb) [reply src/components/notifications/PageNotifications.vue:26]: I don't get that error. The link actually is correct but is blocked somehow as the network doesn't change prior :man-shruggin:

File: src/vuex/modules/session.js [hunk 2]
```diff
@@ -248,9 +248,12 @@ export default ({ apollo }) => {
     getAllSessionsAddresses(store, { networkIds }) {
       let allSessionAddresses = []
       networkIds.forEach((networkId) => {
+        const sessionEntry = localStorage.getItem(`session_${networkId}`)
+        if (!sessionEntry) return
+
         allSessionAddresses.push({
           networkId,
-          address: JSON.parse(localStorage.getItem(`session_${networkId}`))
+          address: JSON.parse(sessionEntry)
```
Comment (Bitcoinera) [inline src/vuex/modules/session.js:256]: 👀

Comment (Bitcoinera) [reply src/vuex/modules/session.js:256]: oh, got it, good!

**Pull request merged**